test/unit/MCR/updateMCR.js

```diff
@@ -44,33 +41,6 @@ describe('updateMCR', function () {
     expect(lastUpdateTime).to.be.equal(previousLastUpdateTime);
   });
 
-  it('keeps values the same if MCR = MCR floor and mcrWithGear is too low', async function () {
```

Comment:
any new unit tests we should add to make sure removing the floor doesn't have weird consequences? they could be added in a later PR too, just worth thinking about it tho

Reply:
Previously it was driven by the maximum of the two, now it's going to be driven by mcr gear alone. Can't see any immediate issues. Please open an issue if this still seems uncertain.